test/unit/prof_hook.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/sh
 
 if [ "x${enable_prof}" = "x1" ] ; then
-  export MALLOC_CONF="prof:true,prof_active:true,lg_prof_sample:0"
+  export MALLOC_CONF="prof:true,prof_active:true,lg_prof_sample:0,prof_bt_max:128"
```

Comment:
Let's add a test case (or maybe an assertion in an existing test) to verify that it's the value we set here.

Reply:
Added an assertion in current test_prof_dump_hook

Reply:
128 happens to be the default value, which isn't the best for testing. Make it 200?